example/src/App.tsx

```diff
@@ -52,9 +57,22 @@ const App = () => {
   const [isNativeAdShowing, setIsNativeAdShowing] = useState(false);
   const [statusText, setStatusText] = useState('Initializing SDK...');
 
+  const appState = useRef(AppState.currentState);
+
   // Run once after mounting
   useEffect(() => {
     initAppLovinMax();
+
+    const subscription = AppState.addEventListener("change", nextAppState => {
+      if (appState.current.match(/inactive|background/) && nextAppState === "active") {
+        showAppOpenAdIfReady();
```

Comment:
Why are we integrating App Open ads in a PR that is to set local extra parameters?

Reply:
Removed.